qa/integration-tests/pom.xml

```diff
@@ -227,6 +227,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
:wrench: Having looked at the implementation of the library, most of the features it adds are not required for us, so imo I would go with just a generic container with the keycloak image and some basic checks, if only to avoid adding more dependencies for little value. Can be a follow up.